samples/getting-started/azure-quantum/Getting-started-with-Cirq-and-Honeywell-on-Azure-Quantum.ipynb

```diff
@@ -0,0 +1,356 @@
+    "\n",
+    "<img src=\"azure-quantum-resource-id.png\" alt=\"How to retrieve the resource ID and location from an Azure Quantum workspace\">"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "da4a3544",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.quantum.cirq import AzureQuantumService\n",
+    "service = AzureQuantumService(\n",
+    "    resource_id=\"\",\n",
+    "    location=\"\",\n",
+    "    default_target=\"honeywell.hqs-lt-s1-apival\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a7a52521",
+   "metadata": {},
+   "source": [
+    "### List all targets\n",
+    "\n",
+    "You can now list all the targets that you have access to, including the current queue time and availability. Running this method will trigger authentication to your Microsoft account, if you're not already logged in."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "ba06b494",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[<Target name=\"ionq.qpu\", avg. queue time=250 s, Available>,\n",
+       " <Target name=\"ionq.simulator\", avg. queue time=0 s, Available>,\n",
+       " <Target name=\"honeywell.hqs-lt-s1\", avg. queue time=0 s, Unavailable>,\n",
+       " <Target name=\"honeywell.hqs-lt-s1-apival\", avg. queue time=0 s, Available>,\n",
+       " <Target name=\"honeywell.hqs-lt-s2\", avg. queue time=93509 s, Available>,\n",
+       " <Target name=\"honeywell.hqs-lt-s2-apival\", avg. queue time=0 s, Available>,\n",
+       " <Target name=\"honeywell.hqs-lt-s1-sim\", avg. queue time=26 s, Available>]"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "service.targets()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b7c41ea3",
+   "metadata": {},
+   "source": [
+    "## Run a simple circuit\n",
+    "\n",
+    "Now, let's create a simple Cirq circuit to run.\n",
+    "\n",
+    "This notebook assumes some familiarity with Cirq. To read more about Cirq, review the [Cirq documentation](https://quantumai.google/cirq)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "defbb444",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<pre style=\"overflow: auto; white-space: pre;\">0: ───H───@───M(&#x27;b&#x27;)───\n",
+       "          │   │\n",
+       "1: ───────X───M────────</pre>"
+      ],
+      "text/plain": [
+       "0: ───H───@───M('b')───\n",
+       "          │   │\n",
+       "1: ───────X───M────────"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import cirq\n",
+    "\n",
+    "q0, q1 = cirq.LineQubit.range(2)\n",
+    "circuit = cirq.Circuit(\n",
+    "    cirq.H(q0), # Hadamard\n",
```

Comment:
How in the heck did Hadamard not get added to cspell yet? Wow.

Reply:
Oops!